workspaces/orchestrator/plugins/orchestrator-form-widgets/README.md

```diff
@@ -68,3 +68,83 @@ Example of response:
   }
 }
 ```
+
+## HTTP Test Server
+
+**For the development purposes only**, there is `http-test-server`, very simple Express Node.js server which responds with JSON schema chunks for the `SchemaUpdater` and other active widgets.
+
+Moreover, it does extensive logging of the received HTTP requests for debugging during the widget's development.
+
+Refer [http-test-server](./http-test-server/README.md) for more details.
```

Comment:
what about renaming it to something like `http-dev-server` while reviewing this PR, I had the feeling that it was intended to be used in tests, but then i realized it's for the `dev` env. Not a strong opinion, just suggesting :) 
If you'll like the suggestion, then please update the references/usage to it accordingly

Reply:
nit: s/Refer [http-test-server]/Refer to [http-test-server]/

Reply:
Thanks, will fix that in #701 which this code is from (will need rebase once #701 gets merged)

Reply:
Will rename the server to `http-workflow-dev-server`, within #701